### ISSUE TITLE
ADA Search Box: Dropdown missing aria-haspopup (Moderate / Easy)

### ISSUE THREAD

**Issue by bbusenius** [uchicago-library/library_website#893]
Part of #324

Deque Issue: 139236

## Problem
- Dropdown missing appropriate aria role

## Solution
- Add aria-haspopup="true"

## Example Code
```html
<button type="button" class="btn dropdown-toggle btn-default" data-toggle="dropdown" title="All Fields" aria-expanded="false"><span class="filter-option pull-left">All Fields</span>&nbsp;<span class="bs-caret"><span class="caret"></span></span></button>
```
